tutorials/segmentation/00_segmentation.py

```diff
@@ -0,0 +1,300 @@
+
+#%%
+# Or it can be used to compute other custom metrics from the segmentation. For
+# instance, the median.
+
+median = np.median(parameters['C_dist_durs'])
+print(f"Microstate C segments have a median duration of {median:.2f}s.")
+
+#%%
+# Transition probabilities
+# ------------------------
+#
+# The obsered transition probabilities (``T``) can be retrieved with the method
+# :meth:`~pycrostates.segmentation.RawSegmentation.compute_transition_matrix`.
+
+T_observed = segmentation.compute_transition_matrix()
+
+#%%
+# This method returns a `~numpy.array` of shape ``(n_clusters, n_clusters)``
+# containing the value corresponding to the chosen statistic:
+#
+# - ``count`` will return the number of transitions observed.
+# - ``probability`` and ``proportion`` will both return the normalized
+#   transition probability.
+# - ``percent`` will return the normalized transition probability as
+#   a percentage.
+#
+# The rows of ``T`` correspond to the ``"from"`` state while the columns
+# correspond to the ``"to"`` state.
+#
+# .. note::
+#
+#     `~numpy.array` are 0-indexed, thus ``T[1, 2]`` encodes the transition
+#     from the second to the third state (``B`` to ``C`` in this tutorial).
+#
+# .. note::
+#
+#     The transition ``from`` and ``to``  unlabeled segments are ignored and
+#     not taken into account during normalization.
+
+ax = sns.heatmap(
+    T_observed,
+    annot=True,
+    cmap="Blues",
+    xticklabels=segmentation.cluster_names,
+    yticklabels=segmentation.cluster_names,
+)
+ax.set_ylabel("From")
+ax.set_xlabel("To")
+
+#%%
+# It is however important to take into account the time coverage of each
+# microstate in order to interpret this matrix. The more a state is
+# present, the more there is a chance that a transition towards this state is
+# observed without reflecting any particular dynamics in state transitions.
+#
+# ``pycrostates`` has the possibility to generate a theoretical transition
+# matrix for a given segmentation with the method
+# :meth:`~pycrostates.segmentation.RawSegmentation.compute_expected_transition_matrix`.
+# This transition matrix is based on segments counts of each state present in
+# the segmentation but ignores the temporal dynamics (effectively randomizing
+# the transition order).
+
+T_expected = segmentation.compute_expected_transition_matrix()
+ax = sns.heatmap(
+    T_expected,
+    annot=True,
+    cmap="Blues",
+    xticklabels=segmentation.cluster_names,
+    yticklabels=segmentation.cluster_names,
+)
+ax.set_ylabel("From")
+ax.set_xlabel("To")
+
+#%%
+# The difference between the observed transition probability matrix
+# ``T_observed`` and the theoretical transition probability matrix
+# ``T_expected`` reveals particular dynamic present in the segmentation.
+# Here we observe that the transition from state ``B`` to state ``A`` appears
+# in larger proportion than expected while the transition from state ``B`` to
+# state ``C`` is less observed than expected.
```

Comment:
@vferat Please double-check this. Previously it was stated from B to C in both cases.

Reply:
all good